src/fetch.js

```diff
@@ -99,19 +99,25 @@ function handleJSONResponse (res) {
   }
 }
 
-export class FetchError {
+export class FetchError extends Error {
   constructor (res, reason) {
+    super()
+    Error.captureStackTrace(this, this.constructor)
```

Comment:
I think `Error.captureStackTrace` is not standard (not available in firefox). We should have a guard `if (Error.captureStackTrace) { .. }`